MAUI/Maps/zoom-pan.md

```diff
@@ -0,0 +1,633 @@
+
+![OSM map zoomlevel](images/zoom-pan/osm_maps_zoomlevel.png)
+
+## Enable zooming 
+
+The zooming feature enables you zoom in and zoom out the maps to show the in-depth information. To enable zooming, set the `EnableZooming` property to `true`. The default value of the `EnableZooming` is `true`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+   <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png" >
+              <map:MapTileLayer.ZoomPanBehavior>
+                  <map:MapZoomPanBehavior ZoomLevel = "2" EnableZooming="True" />
+              </map:MapTileLayer.ZoomPanBehavior>
+        </map:MapTileLayer>
+   </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 2;
+zoomPanBehavior.EnableZooming = true;
+tileLayer.ZoomPanBehavior = zoomPanBehavior;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Enable Panning 
+
+Panning feature allows you move the visible area of the maps when it is zoomed in. To enable panning, set the `EnablePanning` property to `true`. The default value of the `EnablePanning` is `true`.
+
+N> It is applicable for both tile layer and shape layer.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+   <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png" >
+              <map:MapTileLayer.ZoomPanBehavior>
+                  <map:MapZoomPanBehavior ZoomLevel = "2" EnablePanning="True"/>
+              </map:MapTileLayer.ZoomPanBehavior>
+        </map:MapTileLayer>
+   </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 2;
+zoomPanBehavior.EnablePanning = true;
+tileLayer.ZoomPanBehavior = zoomPanBehavior;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customizing min and max zoom level
+
+You can set the min and max zoom level of the map layer by setting the value to `MapZoomPanBehavior.MinZoomLevel` and `MapZoomPanBehavior.MaxZoomLevel` properties. The minimum and maximum zooming levels can be restricted using these properties respectively. The default values of `MapZoomPanBehavior.MinZoomLevel` and `MapZoomPanBehavior.MaxZoomLevel` are 1 and 15 respectively.
+
+However, for `MapTileLayer`, `MapZoomPanBehavior.maxZoomLevel` may slightly vary depends on the providers. Kindly check the respective official website of the map tile providers to know about the maximum zoom level it supports.
```

Comment:
However, for `MapTileLayer,` the `MapZoomPanBehavior.maxZoomLevel` may vary slightly depending on the providers. Kindly check the respective official website of the map tile providers to know about the maximum zoom level it supports.

Reply:
Updated the content